app/assets/javascripts/protocols/index.js

```diff
@@ -209,9 +209,9 @@ function initProtocolPreviewModal() {
           modalTitle.html(data.title);
           modalBody.html(data.html);
           modalFooter.html(data.footer);
-          initHandsOnTable(modalBody);
+          window.initHandsOnTable(modalBody);
```

Comment:
You can just call initHandsOnTable(modalBody) since the problem was that the method hasn't been exposed globaly.